app/code/Magento/Security/Model/AdminSessionsManager.php

```diff
@@ -137,7 +144,7 @@ public function processProlong()
             $this->getCurrentSession()->setData(
                 'updated_at',
                 date(
-                    \Magento\Framework\Stdlib\DateTime::DATETIME_PHP_FORMAT,
+                    DateTime::DATETIME_PHP_FORMAT,
```

Comment:
Can you please add a full class reference here as we are following the same in Core and it's easy to identify class.